src/Php72/Php72.php

```diff
@@ -60,4 +61,23 @@ public static function utf8_decode($s)
 
         return substr($s, 0, $j);
     }
+
+    public static function php_os_family()
+    {
+        if ('\\' === DIRECTORY_SEPARATOR) {
+            return 'Windows';
+        }
+
+        $map = array(
+            'Darwin' => 'Mac',
```

Comment:
this changed recently to Darwin, see http://git.php.net/?p=php-src.git;a=commitdiff;h=a251d9b2c11c2ab53b0214d728c40d70cd06567c;hp=ef4b2fc283ddaf9bd692015f1db6dad52171c3ce